shared/src/hooks/useVaultAccounts.ts

```diff
@@ -4,33 +4,43 @@ import axios from "axios";
 import { BigNumber } from "ethers";
 
 import { impersonateAddress } from "../utils/development";
-import { VaultAddressMap, VaultOptions } from "../constants/constants";
+import {
+  getSubgraphURIForVersion,
+  VaultAddressMap,
+  VaultOptions,
+  VaultVersion,
+} from "../constants/constants";
 import { VaultAccount } from "../models/vault";
-import { getSubgraphqlURI } from "../utils/env";
 import { initialVaultaccounts, useGlobalState } from "../store/store";
 
 const useVaultAccounts = (
   vaults: VaultOptions[],
+  vaultVersions: VaultVersion[],
   {
     poll,
-    pollingFrequency,
+    pollingFrequency = 5000,
   }: {
     poll: boolean;
     pollingFrequency?: number;
-  } = { poll: true, pollingFrequency: 5000 }
+  } = { poll: false, pollingFrequency: 5000 }
 ) => {
   const web3Context = useWeb3React();
   const account = impersonateAddress || web3Context.account;
   const [vaultAccounts, setVaultAccounts] = useGlobalState("vaultAccounts");
   const [loading, setLoading] = useState(false);
 
   const loadVaultAccounts = useCallback(
-    async (vs: VaultOptions[], acc: string, isInterval: boolean = true) => {
+    async (
+      vs: VaultOptions[],
+      vvs: VaultVersion[],
```

Comment:
What's the rationale for using these short variables instead of `vaultOptions` and `vaultVersions`?